packages/tools/client-debugger/client-debugger/src/IContainerDevtools.ts

```diff
@@ -17,8 +17,6 @@ import { AudienceChangeLogEntry, ConnectionStateChangeLogEntry } from "./Logs";
  * Fluid debug session associated with a Fluid Client via its
  * {@link @fluidframework/container-definitions#IContainer} and
  * {@link @fluidframework/container-definitions#IAudience}.
- *
- * @internal
```

Comment:
Is this right? This could stay @internal, no? I also don't see a change for it in the api extractor report.

Reply:
This is no longer exported at all, so it doesn't need the tag anymore. Not really related to this PR, though, so I should probably restore it.